string name
float32 confidence_score
std_msgs/Header header
std_msgs/ColorRGBA color
geometry_msgs/Pose pose
geometry_msgs/Vector3 scale
geometry_msgs/Point max_bound
geometry_msgs/Point min_bound
